README.md

```diff
@@ -67,6 +71,42 @@ Under an automated build environment it makes sense to have a command return a n
 
 ```
 
+### Creating a secure VPN tunnel with ngrok
+
+The CLI has built-in support for [ngrok](https://ngrok.com/) to make it easier for you to run your tests against a locally-accessible application. In order to set up `ngrok` you will need your access token from [your ngrok account](https://dashboard.ngrok.com/get-started/your-authtoken). To initiate a tunnel on execution use the `--ngrokTunnel` parameter to specify your local endpoint, this can be a port on your local computer or a target on the local network:
+
+```
+❯ ghost-inspector test execute <testId> \
+  --ngrokTunnel localhost:8000 \
+  --ngrokToken '<my-ngrok-token>'
+```
+
+**Note**: `--ngrokTunnel` option is not available when using `--immediate`.
+
+If you prefer you can also set the ngrok token using the environment variable `NGROK_TOKEN`.
+
+#### VPN tunnel URL variable
+
+Once you trigger your execution the variable `ngrokUrl` will be made available in your test with the URL of the tunnel. You can modify the name of this variable using the option `--ngrokUrlVariable`, for instance you could set it to `{{ appDomain }}` with the following example:
+
+```
+❯ ghost-inspector test execute <testId> \
+  --ngrokTunnel localhost:8000 \
+  --ngrokToken '<my-ngrok-token>'
+  --ngrokUrlVariable 'appDomain'
+```
+
+You can also set the ngrok URL to the start URL of your test or suite:
+
+```
+❯ ghost-inspector test execute <testId> \
+  --ngrokTunnel localhost:8000 \
+  --ngrokToken '<my-ngrok-token>'
+  --ngrokUrlVariable 'startUrl'
```

Comment:
Since this is a passthrough to [the API endpoint](https://ghostinspector.com/docs/api/tests/#execute), This Should Just Work™